flusher.go

```diff
@@ -16,7 +16,7 @@ import (
 	"time"
 
 	"github.com/DataDog/datadog-go/statsd"
-	"github.com/Sirupsen/logrus"
+	"github.com/sirupsen/logrus"
```

Comment:
?

Reply:
This is so that packages can import sub-packages of logrus, which import logrus with the lower-case name. I guess this won't work because dep pulls in the repo with the upper-case name, so the import fails on linux.

UGH, vendoring, UGH github.